### PR TITLE
[BLOG] PyData Sphinx Theme Accessibility Fixes

### PR DESCRIPTION
**Pull request by gabalafou**
## Preview URL

[Preview: PyData Theme Accessibility Fixes](https://labs-git-feature-pydata-theme-a11y-fixes-quansight.vercel.app/blog/pydata-theme-a11y-fixes)

## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] Blog post featured image is in PNG or JPEG format, **not** SVG.
- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

**Review by gabalafou — COMMENTED** [review]
TODO probably: reduce the size of some of the screenshots

Done self-reviewing.

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (isabela-pf) [inline apps/labs/posts/pydata-theme-a11y-fixes.mdx:99]:
```suggestion
Sphinx-generated docs site. But one of our user-contributors, [Michael Wisely][(https://github.com/michael-wisely-gravwell),
```

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (isabela-pf) [inline apps/labs/posts/pydata-theme-a11y-fixes.mdx:102]:
```suggestion
Then maintainer [Daniel McCloy](https://github.com/drammock) connected the dots. The fix was fortunately
```

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (isabela-pf) [reply apps/labs/posts/pydata-theme-a11y-fixes.mdx:237]: And if he is, I'd link…

```suggestion
expert. Here I would like to a give a big shout out to [Eric Bailey](https://ericwbailey.design/). Some time
```

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (isabela-pf) [reply apps/labs/posts/pydata-theme-a11y-fixes.mdx:294]:
```suggestion
With the help of my designer partner in crime, [Smera Goel](https://github.com/smeragoel), we decided that we
```

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (isabela-pf) [reply apps/labs/posts/pydata-theme-a11y-fixes.mdx:455]: I would hesitate to simply because I don't know what channels of communication we'll maintain. For example, we don't have an email I'm aware of that we are always watching, and I don't know if authors get notified of blog post comments.

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (isabela-pf) [inline apps/labs/posts/pydata-theme-a11y-fixes.mdx:457]: This is a cool section. Even though it felt separate from the rest of the blog post to me, I would keep it. For example, I totally went to check the description for the code and was pleasantly surprised to find Codepens instead of images.

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (isabela-pf) [reply apps/labs/posts/pydata-theme-a11y-fixes.mdx:313]: Other than the existing one? Or are you asking if the existing one makes sense?

While I can't speak from a regular code-writer's perspective, I think having the information represented as code and rendered supports understanding the how behind the before and afters you are showing. This Codepen is also not very long so I don't think it becomes a big obstacle by accident.

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (isabela-pf) [reply apps/labs/posts/pydata-theme-a11y-fixes.mdx:279]: I think the one that is there works for my understanding.

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (isabela-pf) [inline apps/labs/posts/pydata-theme-a11y-fixes.mdx:117]: I liked that you took a moment to provide more, optional information on a screen reader in the last section, so I'm suggest equivalent links in this section. 

```suggestion
This section assumes some knowledge of [HTML](https://en.wikipedia.org/wiki/HTML) and [CSS](https://en.wikipedia.org/wiki/CSS).
```

**Review by isabela-pf — COMMENTED** [review]
A solid blog post with lots of helpful experiential information. I like the way each section is written as a story where readers can witness the original issue, triaging and finding its source, and iterating on how to fix it. I like the honesty and I think it's memorable. 

I wouldn't be surprised to see this posted as-is minus the to-do comments, so I wouldn't block merging on any of my review. Thank you for documenting this process and team wins as a blog post!

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (Carreau) [inline apps/labs/posts/pydata-theme-a11y-fixes.mdx:25]: docs -> documentation ? in a few places maybe. I thing `docs` is colloquial.

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (Carreau) [inline apps/labs/posts/pydata-theme-a11y-fixes.mdx:30]: 2 paragraph starting with `For those not familiar with`.

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (Carreau) [inline apps/labs/posts/pydata-theme-a11y-fixes.mdx:171]: Not for here, but I'm not sre we should have dark code block for a light theme blog.

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (Carreau) [inline apps/labs/posts/pydata-theme-a11y-fixes.mdx:371]: I looked in git history and found no particular reasons.

Comment (Carreau) [reply apps/labs/posts/pydata-theme-a11y-fixes.mdx:371]: ping @choldgraf if he remembers why.

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (smeragoel) [reply apps/labs/posts/pydata-theme-a11y-fixes.mdx:399]: Yes please, go ahead! 😸

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (smeragoel) [inline apps/labs/posts/pydata-theme-a11y-fixes.mdx:22]: This is a small nit-pick, so feel free to ignore if you don't feel the same. As a non-native english speaker, I found the sentence _"For this blog post, I chose...without disabilities, too."_ a bit too long and hard to follow. I think breaking it into two sentences could help.

Something like this maybe:

> For this blog post, I chose three such improvements (out of many!) to examine with a before-and-after lens. This will give a better idea of how these kinds of changes can improve user experience for disabled people—and often
> for people without disabilities, too.

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (trallard) [reply apps/labs/posts/pydata-theme-a11y-fixes.mdx:455]: For PST feedback, this should be the PST repo and issues.
We do not have comments in our blog but there is a connect@quansight.com
email that folks generally use to contact us and we get questions redirected (Ralf and I). So we can do something like

"If you find anything you want to see written about in a blog post like
this one, please let us know! You can contact us at connect@quansight.com."

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (trallard) [inline apps/labs/posts/pydata-theme-a11y-fixes.mdx:465]:
```suggestion
- All images have alt text or are described by the surrounding text (in which case the alt text is an empty string). Even
```
this is the recommended practice and some folks might not know

**Review by trallard — COMMENTED** [review]
Nice one @gab, I think we can leave the blog as is. After re-reading this does not feel too long and there are enough breaks with code samples and images.

**Review by trallard — APPROVED** [review]
Nice one @gabalafou , I think we can leave the blog as is. After re-reading this does not feel too long and there are enough breaks with code samples and images.

File: apps/labs/posts/pydata-theme-a11y-fixes.mdx
Comment (gabalafou) [reply apps/labs/posts/pydata-theme-a11y-fixes.mdx:397]: Mine too :)

**Comment by gabalafou** [comment]
Update: I realized that our blog posts have headings out of order, and since I have a little section at the bottom that talks about that, I want to fix that before merging this PR.

**Review by trallard — COMMENTED** [review]
This looks ready from my POV, but the PR is still in draft so will approve and leave it to @gabalafou  to merge when ready

**Review by trallard — APPROVED** [review]
This looks ready from my POV, but the PR is still in draft so will approve and leave it to @gabalafou  to merge when ready

**Review by trallard — APPROVED** [review]
This looks ready from my POV, but the PR is still in draft so will approve and leave it to @gabalafou  to merge when ready

**Pull request merged**
